docs/search/queries/search-functions/analyzer.md

```diff
@@ -2,75 +2,49 @@
 title: ANALYZER()
 ---
 
-Sets the analyzer for the given search expression. The default analyzer is `identity` for any Search expression. This utility function can be used to wrap a complex expression to set a particular analyzer. It also sets it for all the nested functions which require such an argument to avoid repeating the analyzer parameter. If an analyzer argument is passed to a nested function regardless, then it takes precedence over the analyzer set via `ANALYZER()`.
+Sets an analyzer for the search query. The analyzer applies to all nested functions which do not have an analyzer argument. If a nested function has its own analyzer set, it takes precedence over what you choose here.
 
-`ANALYZER(expr, analyzer)`
+The default analyzer is `identity`.
 
-The `TOKENS()` function is an exception, it requires the analyzer name to be passed in all cases even if wrapped in an `ANALYZER()` call.
+Syntax:
 
-- `expr` (expression): any valid search expression
-- `analyzer` (string): name of an analyzer.
-- returns nothing: The function can only be called in a [search query](../../queries/index.md) and throws an error otherwise.
+`ANALYZER(expr, analyzer)`
 
-Assuming a view definition with an analyzer whose name and type is `delimiter`:
+- `expr` (expression): A valid search expression.
+- `analyzer` (string): Name of the analyzer.
 
-```json
-{
-  "links": {
-    "coll": {
-      "analyzers": [ "delimiter" ],
-      "includeAllFields": true,
-    }
-  },
-  ...
-}
-```
+In this example, assume an analyzer named `delimiter` with the character `|` configured as the delimiter.
 
-… with the analyzer properties `{ "delimiter": "|" }` and an example document `{ "text": "foo|bar|baz" }` in the collection `coll`, the following query would return the document:
+Using the `ANALYZER()` function, you can specify an analyzer type. For example:
 
 ```js
 FOR doc IN viewName
   SEARCH ANALYZER(doc.text == "bar", "delimiter")
   RETURN doc
 ```
 
-The expression `doc.text == "bar"` has to be wrapped by `ANALYZER()` in order to set the analyzer to `delimiter`. Otherwise the expression would be evaluated with the default `identity` analyzer. `"foo|bar|baz" == "bar"` would not match, but the view does not even process the indexed fields with the `identity` analyzer. The following query would also return an empty result because of the analyzer mismatch:
-
-```js
-FOR doc IN viewName
-  SEARCH doc.text == "foo|bar|baz"
-  //SEARCH ANALYZER(doc.text == "foo|bar|baz", "identity")
-  RETURN doc
-```
+The second argument in the ANALYZER() function specifies a `delimiter` analyzer which is configured with the [delimiter property](../../analyzers/properties.md). The expression `doc.text == "bar"` must be wrapped by the `ANALYZER()` function. Otherwise the expression would be evaluated with the default `identity` analyzer. 
```

Comment:
```suggestion
The second argument in the `ANALYZER()` function specifies a `delimiter` analyzer which is configured with the [delimiter property](../../analyzers/properties.md). The expression `doc.text == "bar"` must be wrapped by the `ANALYZER()` function. Otherwise, the expression would be evaluated with the default `identity` analyzer. 
```